CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
-Contributing to Bitcoin Core
-============================
+# Contributing to Bitcoin Classic
 
-The Bitcoin Core project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, testing and patches. This document explains the practical process and guidelines for contributing.
+Bitcoin Classic is hard forking bitcoin to a 2 MB blocksize limit. Please come and join us.
 
-Firstly in terms of structure, there is no particular concept of “Core developers” in the sense of privileged people. Open source often naturally revolves around meritocracy where longer term contributors gain more trust from the developer community. However, some hierarchy is necessary for practical purposes. As such there are repository “maintainers” who are responsible for merging pull requests as well as a “lead maintainer” who is responsible for the release cycle, overall merging, moderation and appointment of maintainers.
+ - [Contribution Guidelines](#contribution-guidelines)
+ - [Communication](#communication)
+ - [Merge Window](#merge-window)
+ - [Consider.it](#considerit)
+ - [Code of Conduct](#codeofconduct)
 
+## <a name="contribution-guidelines"></a> Contribution Guidelines
 
-Contributor Workflow
---------------------
+**Any change to Bitcoin must be respected by 3 groups:**
 
-The codebase is maintained using the “contributor workflow” where everyone without exception contributes patch proposals using “pull requests”. This facilitates social contribution, easy testing and peer review.
+- Developers: which implement changes
+- Miners: which give them computation
+- Users: which give them value
 
-To contribute a patch, the workflow is as follows:
+**We accept pull requests using this rubric:**
 
-  - Fork repository
-  - Create topic branch
-  - Commit patches
+- All pull requests must be within the [merge window](#merge-window).
+- If the pull request has a majority on [consider.it](https://bitcoinclassic.consider.it/). All pull requests must have a vote. A proposal that does not have majority support on [consider.it](#considerit) can be closed at any time by the maintainers.
+- If both Miners and Users want a feature, then we will accept the best pull request for that feature that Developers have implemented
+- If both Miners and Users reject a feature, we will reject pull requests for it.
+- A proposal is in conflict if: (1) Miners and Users disagree on it, or (2) another approved proposal disagrees with it. Github committers can break a tie until Miners and Users resolve their conflicts.
+- If two or more voter-approved pull requests conflict either in code or on concept, then a run-off vote will be held comparing the conflicts.
```

Comment:
That's reasonable. We probably don't need to specify the process for resolving conflicts at this time.

Eventually, it may be possible to use this as a loophole where the maintainer could create a proposal that they know to be in conflict, then use the conflict to give the committer an excuse to do whatever they want on the issue (which could be nothing).... but I think that's more adversarial than we need for the Classic democracy threat model.